src/App.tsx

```diff
@@ -1,14 +1,60 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+
+export function getFilteredTodo(
+  todos: Todo[],
+  filterType: string,
+  query: string,
+) {
+  const filterByType = todos.filter((todo) => {
+    switch (filterType) {
+      case 'active':
```

Comment:
Better create enum for filter type. And use it here `const [filterBy, setFilterBy] = useState('all');` too.